ui/src/organizations/components/OrgTasksPage.tsx

```diff
@@ -95,47 +93,37 @@ class OrgTasksPage extends PureComponent<Props, State> {
 
     return (
       <>
-        <Page titleTag="Tasks">
-          <Input
-            icon={IconFont.Search}
-            placeholder="Filter tasks..."
-            widthPixels={290}
-            value={searchTerm}
-            onChange={this.handleFilterChange}
-            onBlur={this.handleFilterBlur}
-          />
-          <TasksHeader
-            onCreateTask={this.handleCreateTask}
-            setSearchTerm={setSearchTerm}
-            setShowInactive={this.handleToggle}
-            showInactive={showInactive}
-            toggleOverlay={this.handleToggleImportOverlay}
-            showOrgDropdown={false}
-            showFilter={false}
-          />
-          <FilterList<Task>
-            searchTerm={searchTerm}
-            searchKeys={['name', 'labels[].name']}
-            list={this.filteredTasks}
-          >
-            {ts => (
-              <TasksList
-                searchTerm={searchTerm}
-                tasks={ts}
-                totalCount={this.totalTaskCount}
-                onActivate={this.handleActivate}
-                onDelete={this.handleDelete}
-                onCreate={this.handleCreateTask}
-                onClone={this.handleClone}
-                onSelect={this.handleSelectTask}
-                onAddTaskLabels={onAddTaskLabels}
-                onRemoveTaskLabels={onRemoveTaskLabels}
-                onUpdate={this.handleUpdateTask}
-                onRunTask={onRunTask}
-              />
-            )}
-          </FilterList>
-        </Page>
+        <TasksHeader
+          onCreateTask={this.handleCreateTask}
+          setSearchTerm={setSearchTerm}
```

Comment:
Should the searchWidget just be connected. I know in the following labels click update the searchTerm also has to be supplied back to the search widget so it reflects the most recent state of `searchTerm`.

Reply:
This is also the only `searchTerm` connected to the redux store and the rest aren't? Should `SearchWidget` be utilized in those org tabs also?